content/docs/reference/routes/headers.mdx

```diff
@@ -195,9 +195,10 @@ The following token substitutions are available:
 
 | **Token** | **Value** |
 | :-- | :-- |
-| `${pomerium.id_token}` | OIDC ID token from the identity provider\* |
 | `${pomerium.access_token}` | OAuth access token from the identity provider\* |
 | `${pomerium.client_cert_fingerprint}` | Short form SHA-256 fingerprint of the presented client certificate (if [downstream mTLS](/docs/capabilities/mtls-clients) is enabled) |
+| `${pomerium.id_token}` | OIDC ID token from the identity provider\* |
+| `${pomerium.jwt}` | [Pomerium JWT](/docs/capabilities/getting-users-identity) (this is the same value as in the `X-Pomerium-Jwt-Assertion` header, when the [Pass Identity Headers](/docs/reference/routes/pass-identity-headers-per-route) option is enabled) |
```

Comment:
wording here might be slightly confusing, could be misinterpreted to mean that using this requires the pass identity headers option enabled

Reply:
What do you think about this?

Reply:
lgtm